source/tests/compilation/CMakeLists.txt

```diff
@@ -29,3 +35,4 @@ set_target_properties(compilation_test PROPERTIES FOLDER "Tests/Compilation")
 #
 
 add_compilation_test(incompilable)
+add_compilation_test(incompilable_connections)
```

Comment:
rename the tests. incompilable is not true anymore. Also the name is confusingly long and not descriptive: incompilable_connections_not_compilable does not build and incompilable_connections builds.